### DISCUSSION TITLE
Database error saving new user

### DISCUSSION THREAD

**Discussion by matthannam** [supabase/supabase#5949] [Questions]
Hi! I'm working on a project and am stuck on "Database error saving new user" when a new user signs up (not on login).  

This started after restoring a database backup today.  

This error only happens when a **function hook** is enabled.  The function hook posts to an HTTP endpoint when a row is inserted in the `profile`.  I'm following Jon Meyers Stripe / Next methodology here.

Thank you very much for your help!

**Comment by GaryAustin1** [comment]
You might be able to see the error looking in the database logs on the settings page right after you log in.

Also one thing to watch out for using the http extension on auth table inserts/updates is gotrue has a 3 second limit on postgres to return a response or it will error.  So your http call MUST return within 3 seconds.
